### ISSUE TITLE
Note 4 in definition "contrast ratio" still relevant (1.4.3, 1.4.6, 1.4.11)?

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag3#187]
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html#key-terms

I suggest to discuss and check whether note 4 under "Key terms: contrast ratio" is still relevant for 1.4.3, 1.4.6, 1.4.11 and to possibly abolish this note.

> Background color is the specified color of content over which the text is to be rendered in normal usage. It is a failure if no background color is specified when the text color is specified, because the user's default background color is unknown and cannot be evaluated for sufficient contrast. For the same reason, it is a failure if no text color is specified when a background color is specified. 

Reasons: 
* I don't know of any page that meets 1.4.8 (first point: "Foreground and background colors can be selected by the user") in such a way that no colors are defined for text (https://www.w3.org/WAI/WCAG21/Techniques/general/G148), so the technique (which leads to the problem at note 4) is unlikely to be used by visually impaired people.
* In the browsers used by most people today, the colors cannot be set at all (in Chrome, for example). https://www.w3.org/WAI/WCAG21/Techniques/general/G156 mentions IE 7, but who has it? (It still works with IE and Firefox)
* I suppose most visually impaired people use their own user styles or high contrast mode. With both methods the problem of Note 4 does not occur.
* For many people, the note is hard to understand, especially in combination with note 3 immediately before it: "If no background color is specified, then white is assumed."
* I have conducted a survey of accessibility testers in my country and found that almost no one knew about this note 4 and no one tests whether the note is met. It took me many years to find this note myself and to include it in my tests. There are some, but not many pages that violate note 4.

If note 4 should still be relevant, I suggest in WCAG 3.0 to include this directly in the SCs and not to hide it in the Understanding.
